sklearn/cluster/_kmeans.py

```diff
@@ -1240,6 +1240,10 @@ class KMeans(_BaseKMeans):
         If a callable is passed, it should take arguments X, n_clusters and a
         random state and return an initialization.
 
```

Comment:
I think that I would like to use item for the different options above (using `-`). It will make it obvious which part of the discussion is about the options for `init` and which part is to go further (the new link to the example).

Reply:
Hi Guillaume, thank you for reviewing😄
What do you mean by 'using -'?

Reply:
If I understood the original suggestion correctly, I believe it was more intended to separate "If an array is passed..." and "If a callable is passed..." as separate list entries, as opposed to the inputs of the callable that is _now_ listed.

Reply:
Thank you for your reply!
I modified it.

Reply:
Lines underneath a list symbol `*` need to be indented with two spaces (I think. May be four spaces?). The formatting is a bit tricky sometimes. I still struggle with it myself😅

Reply:
Thank you.
I fixed the indentation 😊